src/stores/device/cell.ts

```
@@ -0,0 +1,279 @@
+    } else {
+      this._setCellValue(newValue);
+    }
+  }
+
+  setType(type: CellType) {
+    if (type) {
+      if (cellType.has(type)) {
+        this.type = type;
+      } else {
+        const cellValue = String(this.value).trim().replace(',', '.');
+        const parsedValue = parseFloat(cellValue) || parseInt(cellValue, 10);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)